app/src/test/resources/io/apicurio/registry/rules/validity/xml-schema-invalid-semantics.xsd

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<xs:schema xmlns:xs="http://www.w3.org/2001/XMLSchema">
+<xs:element name="shiporder">
+    <xs:sequence>
+      <xs:element name="orderperson" type="xs:string"/>
+      <xs:element name="shipto">
+        <xs:complexType>
+          <xs:sequence>
+            <xs:element name="name" type="xs:string"/>
+            <xs:element name="address" type="xs:string"/>
+            <xs:element name="city" type="xs:string"/>
+            <xs:element name="country" type="xs:string"/>
+          </xs:sequence>
+        </xs:complexType>
+      </xs:element>
+    <xs:attribute name="orderid" type="xs:string" use="required"/>
+  </xs:complexType>
+</xs:element>
+</xs:schema>
```

Comment:
In what way is this semantically invalid?  I can't tell by just looking. :)

Reply:
trying to remember now! Line 4 has an opening `<xs: sequence>` but its missing closing one. I think the `</xs:complexType>` on line 17 has no opening tag also.

Reply:
OK but those would be syntax errors then, not semantic errors.  So this would fail even if the Validity rule were set to SYNTAX_ONLY

Reply:
A semantic error in the context of XSD would be something that is valid XML syntax but doesn't conform to the XSD format.  Like including an element `<xs:foo>not allowed</xs:foo>` - where the XML is fine but not allowed for an XSD file.

Reply:
Yep your right! Ill need to fix that